docker/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM 		ubuntu:16.04
+
+RUN 		apt-get update
+RUN 		apt-get install -y \
```

Comment:
These should be combined into a single command with `&&`.

Also clean the apt cache after installing packages. See [the apt-get recommendations here](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get).